website/pages/docs/plugins/sources/googleanalytics/_authentication.mdx

```diff
@@ -0,0 +1,53 @@
+[Google Analytics Data API v1](https://developers.google.com/analytics/devguides/reporting/data/v1)
+authentication is based on [OAuth 2.0 authorization](https://developers.google.com/identity/protocols/oauth2).
+
+Two methods are supported: OAuth 2.0 and Application Default Credentials.
+
+### OAuth 2.0
+
+The following options are available when using OAuth:
+
+- **Using an existing access token**
+
+  This token should be authorized for `https://www.googleapis.com/auth/analytics.readonly` scope (e.g. by using [OAuth 2.0 Playground](https://developers.google.com/oauthplayground/)).
+
+- **Using OAuth client ID & client secret**
+
+  You can get your own OAuth credentials using [this guide](https://developers.google.com/identity/protocols/oauth2#1.-obtain-oauth-2.0-credentials-from-the-dynamic_data.setvar.console_name-.).
+
+### Application Default Credentials
+
+See the official [Application Default Credentials guide](https://cloud.google.com/sdk/gcloud/reference/auth/application-default).
+
+**Note**: You will still need to authorize these credentials for `https://www.googleapis.com/auth/analytics.readonly` scope.
+
+Available options are all the same options described [here](https://cloud.google.com/docs/authentication/provide-credentials-adc) in detail.
+
+#### Local Environment
+
+```bash
+gcloud auth application-default login \
+  --scopes=https://www.googleapis.com/auth/analytics.readonly \
+  --client-id-file=[PATH/TO/credentials.json]
+```
+
+See [this guide](https://developers.google.com/analytics/devguides/reporting/data/v1/quickstart-cli) for local environment to get you started.
```

Comment:
```suggestion
See [this guide](https://developers.google.com/analytics/devguides/reporting/data/v1/quickstart-cli) for local environment to get you started.

The final step is to run:

```bash
gcloud auth application-default login \
  --scopes=https://www.googleapis.com/auth/analytics.readonly \
  --client-id-file=[PATH/TO/credentials.json]
```
```